src/cephadm/tests/test_util_funcs.py

```diff
@@ -906,3 +906,65 @@ def test_daemon_sub_id_systemd_names():
     )
     with pytest.raises(ValueError):
         dsi.service_name
+
+
+@pytest.mark.parametrize(
+    "args,new_arg,expected",
+    [
+        (['--foo=77'], '--bar', ['--foo=77', '--bar']),
+        (['--foo=77'], '--foo=12', ['--foo=12']),
+        (
+            ['--foo=77', '--quux=later', '--range=2-5'],
+            '--quux=now',
```

Comment:
Is that the great or the lesser Quux?

Reply:
The lesser northeastern gray quux of course!

Reply:
I figured this was an obscure Guy Steele reference

Reply:
Heh, I had to look that up. Apparently he's associated with that "metasyntactic variable" in particular but I wasn't aware of that off the top of my head. I thought adding lesser/greater to it made it sound like a bird species or something, thus my little joke. :-)